### ISSUE TITLE
[BUG]: Bot shouldn't prompt for Resume style when --resume option is passed

### ISSUE THREAD

**Issue by mrksph** [feder-cr/Auto_Jobs_Applier_AIHawk#681]
### Describe the bug

When the bot is run using the --resume option, this is, when using a CV created by the user instead of generated by the bot it shouldn't ask the user to choose a style.

### Steps to reproduce

1. Run the bot with --resume option
2. Watch the bot ask the user to set a style for resume generation

### Expected behavior

The expected behaviour is that when the --resume option is passed, the bot doesn't need to generate any resume so it shouldn't ask for any styles

### Actual behavior

The bot asks the user to choose between a list of styles

### Branch

main

### Python version

3.10.11

### LLM Used

ChatGPT

### Model used

GPT-4o-mini

### Additional context

This issue seems easy to solve at first, only adding a condition in the main.py file. I would need to open the project in my IDE and debug it to double-check

**Comment by mrksph** [comment]
@feder-cr  are you ok with me opening a PR fixing this?

**Comment by alxmke** [comment]
it doesn't just prompt for style, but seemingly fails to use the provided document at all, and instead tries to generate documents using your LLM API key (at least it does for me, on Windows). another thing: i'm not sure if this is expected behavior, but it scrolls to the bottom of the job results page and back up, and then clicks on one job and then scrolls all the way to the bottom of that job's page and back up too.

running `python main.py --resume data_folder\resume.pdf` gives:

```
2024-10-30 14:25:58.118 | ERROR    | src.llm.llm_manager:__call__:293 - Unexpected error occurred: Error code: 429 - {'error': {'message': 'You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.', 'type': 'insufficient_quota', 'param': None, 'code': 'insufficient_quota'}}
```

**Comment by mrksph** [comment]
> it doesn't just prompt for style, but seemingly fails to use the provided document at all, and instead tries to generate documents using your LLM API key (at least it does for me, on Windows). another thing: i'm not sure if this is expected behavior, but it scrolls to the bottom of the job results page and back up, and then clicks on one job and then scrolls all the way to the bottom of that job's page and back up too.
> 
> running `python main.py --resume data_folder\resume.pdf` gives:
> 
> ```
> 2024-10-30 14:25:58.118 | ERROR    | src.llm.llm_manager:__call__:293 - Unexpected error occurred: Error code: 429 - {'error': {'message': 'You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.', 'type': 'insufficient_quota', 'param': None, 'code': 'insufficient_quota'}}
> ```

Hey I could manage to make it generate custom CVs for me. The README is a bit outdated and it says to use gpt-4o but it fails saying it should be gpt-4o-mini (which is good because it is cheaper). I had to change that in my OpenAI project to make it work.

**Comment by alxmke** [comment]
that's not what my post says. i said it fails to use the user-provided document: `--resume data_folder\resume.pdf `

**Comment by mrksph** [comment]
> that's not what my post says. i said it fails to use the user-provided document: `--resume data_folder\resume.pdf `

Oh, sorry. I thought you meant the plain text resume. In that case I had to provide the full path 

python main.py --resume C:\workspace\Auto_Jobs_Applier_AIHawk\data_folder\My_Resume.pdf

**Comment by mrksph** [comment]
@alxmke hey, did you manage to make it work?

**Comment by 49Simon** [comment]
I will work on this and submit a PR soon

**Comment by cjbbb** [comment]
Fixed by #699 , Thanks 49Simon!
